pom.xml

```diff
@@ -171,7 +171,7 @@
         <commons.collections.version>3.2.2</commons.collections.version>
         <commons.collections4.version>4.3</commons.collections4.version>
 
-        <guava.version>19.0</guava.version>
+        <guava.version>31.0.1-jre</guava.version>
```

Comment:
Please add LICENSE and NOTICE for changed dependencies, refer to this guide: https://inlong.apache.org/community/how-to-maintain-3rd-party-dependencies